API/application/iotjs.py

```diff
@@ -172,6 +180,9 @@ def build(self, device):
         self.__apply_heap_patches(device)
         self.__apply_stack_patches(device)
 
+        if device.get_type() == 'artik053' and device.coverage:
```

Comment:
What about `if device.is_coverage_enabled()`? I think the device should know if the coverage measurement is supported or not.